Documentation/benchmarks/etcd-2-2-0-rc-benchmarks.md

```diff
@@ -20,11 +20,11 @@ Go Version: go1.4.2
 Go OS/Arch: linux/amd64
 ```
 
-Also, we use 3 etcd 2.1.0 alpha-stage members to form cluster to get base performance. etcd's commit head is at [c7146bd5](https://github.com/coreos/etcd/commits/c7146bd5f2c73716091262edc638401bb8229144), which is the same as the one that we use in [etcd 2.1 benchmark](./etcd-2-1-0-benchmarks.md).
+Also, we use 3 etcd 2.1.0 alpha-stage members to form cluster to get base performance. etcd's commit head is at [c7146bd5], which is the same as the one that we use in [etcd 2.1 benchmark][etcd-2.1-benchmark].
 
 ## Testing
 
-Bootstrap another machine and use benchmark tool [boom](https://github.com/rakyll/boom) to send requests to each etcd member. Check [here](../../hack/benchmark/) for instructions.
+Bootstrap another machine and use benchmark tool [boom] to send requests to each etcd member. Check [here][hack-benchmark] for instructions.
```

Comment:
Bootstrap another machine and use the [`boom` HTTP benchmark tool][boom] to send requests to each etcd member. Check the [benchmark hacking guide][hack-benchmark] for detailed instructions.